src/test/java/org/junitpioneer/AbstractPioneerTestEngineTests.java

```diff
@@ -24,10 +24,11 @@
 public abstract class AbstractPioneerTestEngineTests extends AbstractJupiterTestEngineTests {
 
 	protected ExecutionEventRecorder executeTests(Class<?> type, String... methodNames) {
```

Comment:
Would say it's right. I call it this way:

`ExecutionEventRecorder eventRecorder = executeTests(IssueTests.IssueDummyTestClass.class, "testNoAnnotation");`

where `testNoAnnotation`  is the name of a method inside of `IssueDummyTestClass` .

If it would consume _signatures_, you would also have to pass parameters, return value and method modifier (public, static) as they are part of a signature.